src/index.html

```diff
@@ -1,17 +1,196 @@
+            placeholder="email@example.com"
+            tabindex="6"
+          >
+        </div>
+
+        <div class="form__input">
+          <label for="pass">Password:</label>
+          <input
+            type="password"
+            name="pass"
+            id="pass"
+            tabindex="7"
+            minlength="5"
+            required
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="fieldset__opt">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form__input">
+          Do you love cats?
+          <input
+            type="radio"
+            name="lovecat"
+            id="lovecat1"
+            tabindex="8"
+          >
+          <label for="lovecat1">Yes</label>
+          <input
+            type="radio"
+            name="lovecat"
+            id="lovecat2"
+          >
+          <label for="lovecat2">No</label>
+        </div>
+
+        <div class="form__input">
+          <label for="favorite-color">What is your favorite color?</label>
+          <input
+            type="color"
+            name="favorite-color"
+            id="favorite-color"
+            tabindex="9"
+          >
+        </div>
+
+        <div class="form__input">
+          <label for="time-bed">What time do you go to bed?</label>
+          <input
+            type="time"
+            name="time-bed"
+            id="time-bed"
+          >
+        </div>
+
+        <div class="form__input">
+          <label for="favorite-car">What are your favorite brand of cars?</label>
+          <select name="favorite-car"
+            id="favorite-car"
+            multiple tabindex="10"
+          >
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">LADA</option>
+          </select>
+        </div>
+
+        <div class="form__input">
+          <label for="rate-work">How do you rate our work?</label>
+          <input
+            type="range"
+            name="rate-work"
+            id="rate-work"
+            value="0"
+            tabindex="11"
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="fieldset__opt">
+        <legend>Additional info:</legend>
+
+        <div class="form__input">
+          <label for="comments">Comments:</label>
+          <textarea
+          name="comments"
+          id="comments"
+          tabindex="12"
+         ></textarea>
+        </div>
+
+        <div class="form__input">
+          <label for="recommend">Would you recommend us?</label>
+          <select
+            name="recommend"
+            id="recommend"
+            tabindex="13"
+          >
+            <option value="">yes</option>
+            <option value="">no</option>
+          </select>
+        </div>
+      </fieldset>
+      <button type="submit" id="">Submit</button>
+    </form>
     <script type="text/javascript" src="./main.js"></script>
   </body>
-</html>
+</html>
```

Comment:
Always add a neline at the end of each file
```suggestion
</html>

```